src/viur/core/email.py

```diff
@@ -449,3 +452,57 @@ def check_sib_quota() -> None:
             entity["latest_warning_for"] = None
 
         db.Put(entity)
+
+class EmailTransportMailjet(EmailTransport):
+    @staticmethod
+    def splitAddress(address: str) -> dict[str, str]:
+        """
+            Splits a Name/Address Pair into a dict,
+            i.e. "Max Musterman <mm@example.com>" into
+            {"name": "Max Mustermann", "email": "mm@example.com"}
+            :param address: Name/Address pair
+            :return: split dict
+        """
+        posLt = address.rfind("<")
+        posGt = address.rfind(">")
+        if -1 < posLt < posGt:
+            email = address[posLt + 1:posGt]
+            sname = address.replace(f"<{email}>", "", 1).strip()
+            return {"name": sname, "email": email}
+        else:
+            return {"email": address}
+
+    @staticmethod
+    def deliverEmail(*, sender: str, dests: list[str], cc: list[str], bcc: list[str], subject: str, body: str,
+                     headers: dict[str, str], attachments: list[dict[str, bytes]], **kwargs):
+        data = {"messages": [{}]}
+        email = data['messages'][0]
+        email['from'] = EmailTransportMailjet.splitAddress(sender)
+        email['to'] = [EmailTransportMailjet.splitAddress(dest) for dest in dests]
+        email['htmlpart'] = body
+        email['subject'] = subject
+        if bcc:
+            email['bcc'] = [EmailTransportMailjet.splitAddress(b) for b in bcc]
+        if cc:
+            email['cc'] = [EmailTransportMailjet.splitAddress(c) for c in cc]
+        if headers:
+            email['headers'] = headers
+        if attachments:
+            atts = []
+            for att in attachments:
+                next_att = {
+                    "filename": att['filename'],
+                    "base64content": b64encode(att['content']).decode('ASCII')
+                }
+                if 'contenttype' in att:
+                    next_att['contenttype'] = att['contenttype']
```

Comment:
Please use `mimetype` here as described in https://github.com/viur-framework/viur-core/blob/5416176fa03a3492a169517ea3861c8f2b7d3708/src/viur/core/email.py#L178
```suggestion
                if 'mimetype' in att:
                    next_att['contenttype'] = att['mimetype']
```
Btw: I you like you can change the type hint of attachment from `dict` to a [`TypedDict`](https://docs.python.org/3/library/typing.html#typing.TypedDict), this would be more precise.